### PR TITLE
Modernise js

### PR DESCRIPTION
**Pull request by susiecoleman**
Use es6 classes. Defining all functions in the same way even though some functions are only used within the class as opposed to using something like The Revealing Module Pattern so that the code is easier to understand for beginners and appears to have the same functionality as it had previously. Handouts for the education centre will not need to be updated.

### CODE REVIEW

File: src/quiz.js [hunk 1]
```diff
@@ -2,67 +2,69 @@ var quiz = document.querySelector('#quiz');
...
-    this.getSelectedAnswer = function () {
+    getSelectedAnswer() {
         return questionContainer.querySelector(':checked').value;
     }
 
-    this.setQuestion = function (questionText) {
-        var questionElement = getQuestionContainer();
-        setText(questionElement, questionText);
-    };
+    setQuestion(questionText) {
```
Comment (shaundillon) [inline src/quiz.js]: I'd make all these arrow functions just to ensure `this` means what people think it does.

File: src/quiz.js [hunk 2]
```diff
@@ -73,17 +75,17 @@ function Question() {
...
 
     //---------------------------------------------
     //This is the code to change to display all the answer options
 
-    this.setAnswerList = function (answerList) {
-        var answerElement1 = createAnswerContainer();
-        setText(answerElement1, answerList[0]);
-        setRadioButton(answerElement1, 'groupName', answerList[0]);
+    setAnswerList(answerList) {
+        var answerElement1 = this.createAnswerContainer();
```
Comment (shaundillon) [inline src/quiz.js]: If we're doing classes we might as well do `const` and `let`

File: src/quiz.js (hunk 1 above)
Comment (susiecoleman) [reply src/quiz.js]: Discussed offline. Not possible within classes :(

File: src/quiz.js (hunk 2 above)
Comment (susiecoleman) [reply src/quiz.js]: I think we should move to `const` and `let` at some point. However based on the training the education centre staff have had so far I think we are best to stick to `var` which they are already familiar with. Also changing to `const` and `let` requires changes to the education centre handouts and additional work for them.

**Pull request merged**